### DISCUSSION TITLE
Welcome to gltfjsx Discussions!

### DISCUSSION THREAD

**Discussion by drcmda** [pmndrs/gltfjsx#98] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by teenkwn** [comment]
Hello, @drcmda I like this lib so much but I would like to implement this func into my react native application
I've already try it but it fail.
Is that support? Do you have any suggestion on this problem?
